### PR TITLE
Update setup guides in core docs to include steps to connect to a local devnet

### LINKED ISSUE #786: Update setup guides to include steps to connect to a local Devnet
### What
Setup guides for Lisk Core

### How

- Update setup guides with instructions on how to connect to a devnet with Lisk Core.
- Add info about the Genesis account.

### Additional information

you can use lisk-core start --network devnet as you mentioned.
The genesis block passphrase is `peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready`

### Page location 
setup/index.adoc

### Affected branch
`release/4.0.0`

### PR DESCRIPTION
**Pull request by chris529**
Closes #786

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/setup/binary.adoc [hunk 1]
```diff
@@ -303,6 +304,15 @@ Testnet::
...
 --
+Devnet::
++
+--
+[source,bash]
+----
+lisk-core start --network devnet
+----
+--
+NOTE: The passphrase for the genesis account can be found here in the Lisk Core GitHub repository under the following path: xref:{url_devnet_passphrase}[config/devnet/passphrase.json].
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/binary.adoc:315]:
```suggestion
NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: xref:{url_devnet_passphrase}[config/devnet/passphrase.json].
```

File: docs/antora/modules/ROOT/pages/setup/npm.adoc [hunk 2]
```diff
@@ -192,6 +193,16 @@ Testnet::
...
 --
+Devnet::
++
+--
+[source,bash]
+----
+lisk-core start --network devnet
+----
+--
+NOTE: The passphrase for the genesis account can be found here in the Lisk Core GitHub repository under the following path: xref:{url_devnet_passphrase}[config/devnet/passphrase.json].
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/npm.adoc:204]:
```suggestion
NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: xref:{url_devnet_passphrase}[config/devnet/passphrase.json].
```

**Review by sameersubudhi — APPROVED** [review]

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 3]
```diff
@@ -43,7 +43,7 @@ include::partial$create-a-new-user.adoc[]
 To run Lisk Core in Docker, firstly it is necessary to install the Docker engine.
 Determine if your platform can run Docker as described below.
 
-Please refer to {url_docker_install}[https://docs.docker.com/engine/installation]
+Please refer to {url_docker_install}[https://docs.docker.com/engine/installation].
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:46]: Please use the variable convention for links

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 4]
```diff
@@ -160,7 +160,21 @@ docker run --volume lisk-data:/home/lisk/.lisk \
...
+[source,bash]
+----
+docker run --volume lisk-data:/home/lisk/.lisk \
+           --publish 7887:7887 \
+           --name lisk-core \
+           lisk/core:4.0.0 \
+           start --network=devnet
+----
 --
+NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: xref:{url_devnet_passphrase}[config/devnet/passphrase.json].
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:177]:
```suggestion
NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: {url_devnet_passphrase}[config/devnet/passphrase.json].
```

File: docs/antora/modules/ROOT/pages/setup/npm.adoc [hunk 5]
```diff
@@ -192,6 +193,16 @@ Testnet::
...
 --
+Devnet::
++
+--
+[source,bash]
+----
+lisk-core start --network devnet
+----
+--
+NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: xref:{url_devnet_passphrase}[config/devnet/passphrase.json].
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/npm.adoc:204]:
```suggestion
NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: {url_devnet_passphrase}[config/devnet/passphrase.json].
```

File: docs/antora/modules/ROOT/pages/setup/snap.adoc [hunk 6]
```diff
@@ -83,5 +84,14 @@ Testnet::
...
 --
+Devnet::
++
+--
+[source,bash]
+----
+lisk-core start --network devnet
+----
+--
+NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: xref:{url_devnet_passphrase}[config/devnet/passphrase.json].
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/snap.adoc:95]:
```suggestion
NOTE: The passphrase for the genesis account(s) can be found here in the Lisk Core GitHub repository under the following path: {url_devnet_passphrase}[config/devnet/passphrase.json].
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

**Review by TalhaMaliktz — APPROVED** [review]
Thank you! LGTM 👍🏼

**Pull request merged**